.doctrine-project.json

```diff
@@ -3,6 +3,12 @@
     "shortName": "DoctrineMongoDBBundle",
     "slug": "doctrine-mongodb-bundle",
     "versions": [
+        {
+            "name": "4.4",
+            "branchName": "4.4.x",
```

Comment:
Shouldn't this happen after we release 4.3.0 and have 4.4.x branch in place?

Reply:
`master` can be renamed to 4.4.x right now, there is no need for releasing 4.3.0 for that, is there? It's the first item of the todo list :)

Reply:
Dang, this is about renaming master to 4.4.x first 🤦‍♂️

Reply:
@greg0ire yeah sorry, I went straight to the review from first email and haven't seen the todo list :)

Reply:
@malarzm let's proceed? I can do it if you don't feel comfortable with that, but I'd rather you approve this PR before going through the todo list myself.

Reply:
Sorry for the lag, sadly I didn't find as much time for ODM last week as I wanted. Going through the checklist now :)